Fixes #12365 - Recognize virtual interfaces with alphanum aliases as such

roidelapluie opened the pull request:
We have in our lab virtual interfaces with the login of the developers
(e.g eth0:roidelapluie). They are recognized as physical interface and
overwrite eth0.

Expected in the foreman:
eth0 phy
eth0_roidelapluie virt
eth0_julien virt

got:
eth0_julien phy

This commit fixes that by changing the regex that checks for virtual
interfaces.

roidelapluie commented:
@domcleal blocked due to broken katello tests (independent from this patch)

dLobatog commented:
Yeah, https://github.com/Katello/katello/pull/5561 should have fixed it - [test]

Commit:
Fixes #12365 - Recognize virtual interfaces with alphanum aliases as such

We have in our lab virtual interfaces with the login of the developers
(e.g eth0:roidelapluie). They are recognized as physical interface and
overwrite eth0.

Expected in the foreman:
eth0 phy
eth0_roidelapluie virt
eth0_julien virt

got:
eth0_julien phy

This commit fixes that by changing the regex that checks for virtual
interfaces.

dLobatog commented:
Merged as 8adab989ded287bd4453a6b60ab00a682f29187a, thanks @roidelapluie!

dLobatog commented:
86701faf31a16925a6c689d1bcd6630c655dada7
